### PR TITLE
Adds a Random Forrest classifier ScorerModel with tests.

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

File: revscoring/scorers/tests/test_rf.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from ..rf import RFModel
+from .util import FEATURES, pickle_and_unpickle, train_score
+
+
+def test_rf():
+    model = RFModel(FEATURES)
+    train_score(model)
```
Comment (he7d3r) [inline revscoring/scorers/tests/test_rf.py]:
```python
$nosetests
...................................................................................................................F.......
======================================================================
FAIL: revscoring.scorers.tests.test_rf.test_rf
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/helder/env/3.4/lib/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/helder/projects/revscoring/revscoring/scorers/tests/test_rf.py", line 7, in test_rf
    train_score(model)
  File "/home/helder/projects/revscoring/revscoring/scorers/tests/util.py", line 42, in train_score
    eq_(score_doc['prediction'], True)
AssertionError: False != True

----------------------------------------------------------------------
Ran 123 tests in 8.707s

FAILED (failures=1)
```

File: revscoring/scorers/tests/util.py [hunk 2]
```diff
@@ -1,61 +1,63 @@
...
+            (True for i in range(50)))
+    ))
+    deterministic.shuffle(observations)
+
+    mid = int(len(observations)/2)
+    train_set = observations[:mid]
+    test_set = observations[mid:]
+
+    model.train(train_set)
+    score_doc = model.score((-1,-2))
```
Comment (halfak) [inline revscoring/scorers/tests/util.py]: Meaningful change is here.

File: revscoring/scorers/tests/util.py [hunk 3]
```diff
@@ -1,61 +1,63 @@
...
+
+    print(score_doc['probability'])
```
Comment (he7d3r) [inline revscoring/scorers/tests/util.py]: Stray print?

File: revscoring/scorers/tests/util.py (hunk 2 above)
Comment (he7d3r) [inline revscoring/scorers/tests/util.py]: The previous error is gone, but I'm getting a new one, which does not happens on master:
```python
$nosetests
.................................................................................................................E.........
======================================================================
ERROR: revscoring.scorers.tests.test_nb.test_multinomial_nb
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/helder/env/3.4/lib/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/helder/projects/revscoring/revscoring/scorers/tests/test_nb.py", line 13, in test_multinomial_nb
    pickle_and_unpickle(model)
  File "/home/helder/projects/revscoring/revscoring/scorers/tests/util.py", line 63, in pickle_and_unpickle
    train_score(reconstructed_model)
  File "/home/helder/projects/revscoring/revscoring/scorers/tests/util.py", line 39, in train_score
    model.train(train_set)
  File "/home/helder/projects/revscoring/revscoring/scorers/scorer.py", line 237, in train
    self.classifier_model.fit(values, labels)
  File "/home/helder/env/3.4/lib/python3.4/site-packages/sklearn/naive_bayes.py", line 497, in fit
    self._count(X, Y)
  File "/home/helder/env/3.4/lib/python3.4/site-packages/sklearn/naive_bayes.py", line 598, in _count
    raise ValueError("Input X must be non-negative")
ValueError: Input X must be non-negative

----------------------------------------------------------------------
Ran 123 tests in 3.862s

FAILED (errors=1)
```

File: revscoring/scorers/tests/util.py (hunk 3 above)
Comment (halfak) [inline revscoring/scorers/tests/util.py]: Removed in next commit.

File: revscoring/scorers/tests/util.py (hunk 2 above)
Comment (halfak) [inline revscoring/scorers/tests/util.py]: That is due to some weirdness in the multinomial Naive Bayes classifier.  See https://github.com/halfak/Revision-Scoring/issues/65

**Pull request merged**